hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/conf/OzoneConfiguration.java

```diff
@@ -335,4 +334,5 @@ private static void addDeprecatedKeys() {
             OZONE_CONTAINER_COPY_WORKDIR)
     });
   }
+
```

Comment:
Can you revert the changes in this file since it's actually doesn't change any content?

Reply:
Done.